doc/languages-frameworks/python.section.md

```diff
@@ -976,7 +1093,7 @@ example of such a situation is when `py.test` is used.
 #### Common issues
 
 * Non-working tests can often be deselected. By default `buildPythonPackage`
-  runs `python setup.py test`. Most python modules follows the standard test
+  runs `python setup.py test`. Most Python modules follows the standard test
   protocol where the pytest runner can be used instead. `py.test` supports a
   `-k` parameter to ignore test methods or classes:
```

Comment:
Additionally, I'd note that in some cases it's fine to set `doCheck` to `false` (e.g. if there are no tests or only impure tests), but in that case a comment with a rationale should be left above the `doCheck`-assignment.

Reply:
I think that's mentioned in the contributing part.